docs/references/javascript/clerk/create-organization.mdx

```diff
@@ -243,7 +243,7 @@ All props below are optional.
 | Name | Type | Description |
 | --- | --- | --- |
 | `afterCreateOrganizationUrl` | `string` | Full URL or path to navigate after creating a new organization. |
-| `routing` | `'hash' \| 'path' \| 'virtual'` | The routing strategy for your pages. <br />Supported values are: <ul><li>`hash` (default): Hash-based routing.</li> <li>`path`: Path-based routing.</li> <li>`virtual`: Virtual-based routing.</li></ul> |
+| `routing` | `'hash' \| 'path' \| 'virtual'` | The routing strategy for your pages.<br />Note: If you are using environment variables for Next.js or Remix to specify your routes, this will be set to `path`. |

```

Comment:
I don't think this note is applicable to to this component. AFAIK only sign in and sign up are impacted.